### DISCUSSION TITLE
Remove Recipe?

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#1466] [Help] [answered]
Is it possible to remove a recipe from the recipe book of a player after it has been added? And if so, how?

**Comment by GreatWyrm** [comment, marked as answer]
You will need to send an [Unlock Recipes Packet](https://javadoc.minestom.net/net/minestom/server/network/packet/server/play/UnlockRecipesPacket.html) with mode 2 to remove recipes. Details of how the packet works can be found [here](https://wiki.vg/Protocol#Update_Recipe_Book).

**Reply by KyllianGamer** [reply]
Thanks!
